recipes/cronjob/config.cron.yml

```diff
@@ -0,0 +1,7 @@
+webimage_extra_packages: [--no-install-recommends cron]
+
+hooks:
+  post-start:
+    - exec: echo '*/1 * * * * root TYPO3_CONTEXT=Development/Local /usr/bin/php /var/www/html/public/typo3/sysext/core/bin/typo3 scheduler:run' | sudo tee -a /etc/cron.d/typo3
```

Comment:
I think it would probably be cleaner to do these all in one step, or at least do the chmod and start in one step.